### DISCUSSION TITLE
About training the StyleGAN V2 in GLEAN

### DISCUSSION THREAD

**Discussion by tongzheng-dev** [open-mmlab/mmediting#1209] [Models] [answered]
Hi,
seems you provide serval pre-trained StyleGAN V2 models for GLEAN. But what if I want to train a StyleGAN V2 on my own dataset? Does mmeediting support trainig of StyleGAN V2?

**Comment by ckkelvinchan** [comment, marked as answer]
You need to use [MMGeneration](https://github.com/open-mmlab/mmgeneration) to train StyleGAN. After training, you can directly use the checkpoint in GLEAN.
